client/my-sites/checklist/checklist-show/index.jsx

```diff
@@ -29,11 +29,17 @@ class ChecklistShow extends PureComponent {
 		this.props.loadTrackingTool( 'HotJar' );
 	}
 
+	isComplete( taskId ) {
+		return get( this.props.taskStatuses, [ taskId, 'completed' ], false );
+	}
+
 	handleTaskStart = task => () => {
 		const { requestTour, siteSlug, track } = this.props;
-
 		launchTask( {
-			task,
+			task: {
+				...task,
+				completed: task.completed || this.isComplete( task.id ),
```

Comment:
Task completion may come from static declaration or app state. Internally, launchTask will check `task.completed`.

`launchTask` _should_ disappear in the near future 🤞